### PR TITLE
Feature/adding user controller PLEASE CHECK THIS

### PR DESCRIPTION
**Pull request by elcapitan89**

### CODE REVIEW

File: server/controllers/gainzController.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+const db = require('../server/postgresPool.js');
+
+const gainzController = {};
+
+gainzController.getExcercises = (req, res, next) => {
```
Comment (labaird) [inline server/controllers/gainzController.js]: please change getExcersises to getExercise

File: server/controllers/gainzController.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  const exerciseQ = 'SELECT DATA FROM EXERCISE TABLE';
+  db.query(exerciseQ)
+    .then(data => {
+      res.locals.excerciseQuery = data.rows;
```
Comment (labaird) [inline server/controllers/gainzController.js]: change excercise to exercise

File: server/controllers/gainzController.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    })
+    .catch(err => {
+      console.log(err);
+      next();
+    });
+};
+
+
+gainzController.getHistory = (req, res, next) => {
+  const drillQuery = 'SELECT DATA FROM DRILL TABLE';
```
Comment (labaird) [inline server/controllers/gainzController.js]: Not proper DB query. Please review docs

File: server/controllers/gainzController.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  db.query(drillQuery)
+    .then(data => {
+      res.locals.drillQuery = data.rows[0];
```
Comment (labaird) [inline server/controllers/gainzController.js]: Why only grabbing one data item? Don't we need an array of all items for history?

File: server/routes/api.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+
+const gainzController = require('../controllers/gainzController');
+
+const router = express.Router();
+
+router.get('/',
+  gainzController.getExcercises,
```
Comment (labaird) [inline server/routes/api.js]: change excercises to exercises - spellcheck here too!

File: server/routes/api.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+//   (req, res) => res.status(200).json(res.locals.homeQuery),
+// );
+
+// router.get('/login',
+//   gainzController.getLogin,
+//   (req, res) => res.status(200).json({}),
+// );
+
+// router.get('/logout',
+//   gainzController.getFilm,
```
Comment (labaird) [inline server/routes/api.js:28]: Why is getFilm here? Please don't copy-paste. You will grow as a developer if you can type this yourself from memory!

File: server/routes/api.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+//   (req, res) => res.status(200).json({}),
+// );
+
+// router.get('/signup',
+//   gainzController.getFilm,
```
Comment (labaird) [inline server/routes/api.js:33]: No getFIlm

File: server/routes/api.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+  (req, res) => res.status(200).json(res.locals.excerciseQuery),
+);
+
+router.get('/history',
+  gainzController.getHistory,
+  (req, res) => res.status(200).json(res.locals.drillQuery),
+);
+
+// router.get('/createExercise',
+//   gainzController.getNewExcercises,
```
Comment (labaird) [inline server/routes/api.js]: change excercises to exercises

**Review by labaird — COMMENTED** [review]
Spelling errors and some of the commands to the DB are very odd! Please work with Paul tonight to review a solution!

**Review by PLCoster — APPROVED** [review]
Looking good postman routes working

**Pull request merged**